docs/docs/tutorial.md

```diff
@@ -308,6 +308,18 @@ Here we tell the interpreter that we want to relate to an `Employee` node with a
 nodestream to know which `Employee` node to relate to, we need to specify the key of the related node. In our case,
 we can do that by extracting the value of `reports_to`  and mapping it to the `employee_id` key.
 
+## Managing the Database 
+
+`nodestream` uses the concept of migrations to manage the schema and state of the database. Every time you make a change
+to the schema of the data, you need to make and run migrations. To do so, run:
+
+```bash
+nodestream migratations make
+```
+
+This will create a new `.yaml` migration file in the `migrations` directory. Each migration file is named with a
+timestamp and (sometimes) a description of the migration. For example:
+
```

Comment:
Come back here and fill out sample output from this command. (Basically follow the tutorial from start to finish)